server.py

```diff
@@ -0,0 +1,55 @@
+import chat_proto.chat_pb2_grpc as chat_pb2_grpc
+import chat_proto.chat_pb2 as chat_pb2
+from concurrent import futures
+import grpc
+import time
+
+
+class ChatServiceServicer(chat_pb2_grpc.ChatServiceServicer):
+    
+
+    list_of_users = chat_pb2.Users()
+
+    list_of_messages = []
+
+
+    def sendMessage(self, request, context):
+
+        message = request.message
+        message.created_at = time.time()
+        ChatServiceServicer.list_of_messages.append(message)
+
+        # Check users if they exist in our system
+        if message.from_user not in ChatServiceServicer.list_of_users.user:
+            ChatServiceServicer.list_of_users.user.add(login=message.from_user.login)
+
+        if message.to_user not in ChatServiceServicer.list_of_users.user:
+            ChatServiceServicer.list_of_users.user.add(login=message.to_user.login)
+
+
+        return chat_pb2.sendMessageResponce()
+    
+
+    def getUsers(self, request, context):
+            
+        return chat_pb2.getUsersResponce(users=ChatServiceServicer.list_of_users)
+    
+
+    def getMessages(self, request, context):
+        
+        user_login = request.user.login
+
+        for message in ChatServiceServicer.list_of_messages:
```

Comment:
I would suggest not using `list` of messages. It's better to use `dict` where each key is a user login. Is such a way don't need to iterate over all messages  every time, you just return all messages by key